src/condor_utils/submit_utils.cpp

```diff
@@ -467,7 +468,7 @@ void SubmitHash::setup_submit_time_defaults(time_t stime)
 	sv->psz = &times[8];
 
 	// set SUBMIT_TIME macro value
-	sprintf(&times[12], "%lu", (unsigned long)stime);
+	{ auto [p, ec] = std::to_chars(&times[12], &times[36], (unsigned long) stime); *p = '\0';}
```

Comment:
Shouldn't it be this:
```suggestion
	{ auto [p, ec] = std::to_chars(&times[12], &times[23], (unsigned long) stime); *p = '\0';}
```

Reply:
times is 24 bytes and our one-past-the-end needs to be within the array for writing the NUL.